handbook/product/README.md

```diff
@@ -16,18 +16,37 @@ The product team is responsible for product design tasks like drafting [changes
 
 At Fleet, like [GitLab](https://about.gitlab.com/handbook/product-development-flow/#but-wait-isnt-this-waterfall) and [other organizations](https://speakerdeck.com/mikermcneil/i-love-apis), every change to the product's UI gets [wireframed first](https://fleetdm.com/handbook/company/why-this-way#why-do-we-use-a-wireframe-first-approach).
 
-* Take the top issue that is assigned to you in the "Prioritized" column of the drafting board.
+- Take the top issue that is assigned to you in the "Prioritized" column of the drafting board.
+  
+- Create a file inside the [Fleet product Figma project](https://www.figma.com/files/project/17318630/%F0%9F%94%9C%F0%9F%93%A6-Fleet-EE%C2%AE-(product)?fuid=1234929285759903870).
+  -  Duplicate "(TEMPLATE) Starter file". It's pinned on the top of the "Fleet product" project.
+  -  Each Figma file corresponds to an issue on GitHub. Rename the file to follow the naming convention: **{issue number} {issue name}** (e.g. #11766 Instructions for Autopilot enrollment).
+  -  Starter file will have 4 predefined pages: Cover, Explorations, Archive, and Ready for development
+    -  Cover page: This page has a component with issue number, issue name, and status fields. There are 3 statuses: Work in progress, Approved, and Released (the main source of truth is still the drafting board)
+    -  Explorations page: use to explore and review wireframes.
+    -  Archive page: use to move all screens that are not relevant for design reviews and might be useful in the future.
+    -  Ready for development: Once ready for development move all relevant screens to this page. This way you can still keep your explorations and revisit them at some point in the future if necessary.
```

Comment:
Had some feedback about the structure that I didn't communicate very well verbally the other day, so made a temporary copy of the template file and added my suggestions [there](https://www.figma.com/file/sDGpOdIC3rvpmHUeLlKE4J/TEMPORARY-COPY-(suggestions)?type=design&node-id=0%3A1&mode=design&t=zMnXBrz9EOgtyykt-1).

Basically, my thoughts around this are: the intended audience for these wireframes is @mikermcneil at design reviews (only looking @ what the designer assumes is ready for dev, we don't show him WIP/alternate versions), and the engineering team (after the designs have been approved), so it makes sense to have the section aimed at them listed first.

Anything else is leftovers, kept around for convenience, and I'd advocate for making that as clear (and separate) as possible.

With that in mind, here's the structure I'd suggest:
<img width="408" alt="Screenshot 2023-07-05 at 5 40 31 PM" src="https://github.com/fleetdm/fleet/assets/3065949/ddeb11c3-5caf-45f9-9397-ba02aba1b10e">

Reply:
> Had some feedback about the structure that I didn't communicate very well verbally the other day, so made a temporary copy of the template file and added my suggestions [there](https://www.figma.com/file/sDGpOdIC3rvpmHUeLlKE4J/TEMPORARY-COPY-(suggestions)?type=design&node-id=0%3A1&mode=design&t=zMnXBrz9EOgtyykt-1).
> 
> Basically, my thoughts around this are: the intended audience for these wireframes is @mikermcneil at design reviews (only looking @ what the designer assumes is ready for dev, we don't show him WIP/alternate versions), and the engineering team (after the designs have been approved), so it makes sense to have the section aimed at them listed first.
> 
> Anything else is leftovers, kept around for convenience, and I'd advocate for making that as clear (and separate) as possible.
> 
> With that in mind, here's the structure I'd suggest: <img alt="Screenshot 2023-07-05 at 7 36 57 PM" width="401" src="https://user-images.githubusercontent.com/3065949/251311978-237433d9-cb3c-438f-9869-00db96c458a4.png">

@rachaelshaw That makes sense. I thought we could have "--- NOT FOR DEVELOPMENT ---" instead of "--- ARCHIVE BELOW ---". That would make it clear it's not for development and we could have multiple pages below that divider if necessary.